api/XMLHttpRequest.json

```diff
@@ -0,0 +1,1610 @@
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        }
+      },
+      "setRequestHeader": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/XMLHttpRequest/setRequestHeader",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": [
+              {
+                "version_added": "7"
+              },
+              {
+                "notes": "Implemented via <code>ActiveXObject</code>",
+                "version_added": "5"
+              }
+            ],
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "1.2"
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "sendAsBinary": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/XMLHttpRequest/sendAsBinary",
+          "support": {
+            "webview_android": {
+              "notes": "There is a <a href='https://developer.mozilla.org/docs/Web/API/XMLHttpRequest#sendAsBinary%28%29_polyfill'>polyfill available</a> to support <code>sendAsBinary()</code>.",
+              "version_added": false
+            },
+            "chrome": {
+              "notes": "There is a <a href='https://developer.mozilla.org/docs/Web/API/XMLHttpRequest#sendAsBinary%28%29_polyfill'>polyfill available</a> to support <code>sendAsBinary()</code>.",
+              "version_added": false
+            },
+            "chrome_android": {
+              "notes": "There is a <a href='https://developer.mozilla.org/docs/Web/API/XMLHttpRequest#sendAsBinary%28%29_polyfill'>polyfill available</a> to support <code>sendAsBinary()</code>.",
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": [
+              {
+                "version_removed": "31"
```

Comment:
Here lies the travis failure: the two blocks needs to be one. Like this:
{
 version_added: 2,
 version_removed: 31
}